v0/destinations/woopra/data/WoopraIdentifyConfig.json

```diff
@@ -0,0 +1,282 @@
+[
+    {
+        "sourceKeys": [
+            "traits.email",
+            "context.traits.email",
+            "properties.email"
+        ],
+        "destKey": "cv_email"
+    },
+    {
+        "sourceKeys": "title",
+        "destKey": "cv_title",
+        "sourceFromGenericMap": true,
+        "metadata": {
+            "strictMultiMap": [
+                {
+                    "sourceVal": [
+                        "Mr",
+                        "Mr.",
+                        "mr",
+                        "MR"
+                    ],
+                    "destVal": "Mr"
+                },
+                {
+                    "sourceVal": [
+                        "Mrs",
+                        "Mrs.",
+                        "mrs",
+                        "MRS"
+                    ],
+                    "destVal": "Mrs"
+                },
+                {
+                    "sourceVal": [
+                        "Miss",
+                        "Miss.",
+                        "miss",
+                        "MISS"
+                    ],
+                    "destVal": "Miss"
+                }
+            ]
+        }
+    },
+    {
+        "sourceKeys": "createdAt",
```

Comment:
Can you please clarify why we have chosen createdAt rather than originalTimestamp?